Framework/Kernel/src/DirectoryValidator.cpp

```diff
@@ -7,8 +7,8 @@
 #include "MantidKernel/DirectoryValidator.h"
 #include "MantidKernel/IValidator.h"
 #include <Poco/Exception.h>
-#include <Poco/File.h>
 #include <Poco/Path.h>
```

Comment:
This header is not required here

Reply:
I was unable to find a replacement for `Poco::Path().tryParse(value)`